app/controllers/api/v1/flavors_controller.rb

```diff
@@ -1,6 +1,7 @@
 class Api::V1::FlavorsController < ApplicationController  
   def show
     @flavor = Flavor.find(params[:id])
-    render json: @flavor
+    @reviews = @flavor.reviews
+    render json: { flavorData: @flavor, reviewsData: @reviews }
```

Comment:
ill allow this, but it would be better with a serializer!!

Reply:
We'll keep this in mind!